DSharpPlus/Entities/DiscordAuditLogObjects.cs

```diff
@@ -560,6 +570,103 @@ public sealed class DiscordAuditLogIntegrationEntry : DiscordAuditLogEntry
         public PropertyChange<int?> ExpireBehavior { get; internal set; }
     }
 
+    public sealed class DiscordAuditLogGuildScheduledEventEntry : DiscordAuditLogEntry
+    {
+        /// <summary>
+        /// Gets a change in the event's name
+        /// </summary>
+        public PropertyChange<string?> Name { get; internal set; }
+
+        /// <summary>
+        /// Gets the target event. Note that this will only have the ID specified if it is not cached.
+        /// </summary>
+        public DiscordScheduledGuildEvent Target { get; internal set; }
+
+        /// <summary>
+        /// Gets the channel the event was changed to.
+        /// </summary>
+        public PropertyChange<DiscordChannel?> Channel { get; internal set; }
+
+        /// <summary>
+        /// Gets the description change of the event.
+        /// </summary>
+        public PropertyChange<string?> Description { get; internal set; }
+
+        /// <summary>
+        /// Gets the change of type for the event.
+        /// </summary>
+        public PropertyChange<ScheduledGuildEventType?> Type { get; internal set; }
+
+        /// <summary>
+        /// Gets the change in image hash.
+        /// </summary>
+        public PropertyChange<string?> ImageHash { get; internal set; }
+
+        /// <summary>
+        /// Gets the change in event location, if it's an external event.
+        /// </summary>
+        public PropertyChange<string?> Location { get; internal set; }
+
+        /// <summary>
+        /// Gets change in privacy level.
+        /// </summary>
+        public PropertyChange<ScheduledGuildEventPrivacyLevel?> PrivacyLevel { get; internal set; }
```

Comment:
Is this on the docs? Technically speaking this never changes because the only available privacy level is GUILD_ONLY (rip stage instances)

Reply:
Yeah this is in the docs. Could just ignore it if it's not going to change

Reply:
Interesting. It'd be fine with or without, frankly. Probably best to keep it in in case Discord decides to change their mind.